docs/api/js/utils.md

```diff
@@ -38,17 +43,35 @@ export const IERC20 = new utils.Interface(require("../abi/IERC20.json"));
 Utility method that returns the correctly formed `paymasterParams` object for the common [paymaster flows](../../dev/developer-guides/aa.md#built-in-paymaster-flows).
 
 ```typescript
-export function getPaymasterParams(paymasterAddress: Address, paymasterInput: PaymasterInput): PaymasterParams;
+export function getPaymasterParams(paymasterAddress: Address, paymasterInput: PaymasterInput): PaymasterParams {
```

Comment:
I'd say that implementation details are redundant here. the definition is enough